README.md

```diff
@@ -132,6 +137,126 @@ cargo generate -i ~/.ssh/id_ed25519 https://github.com/fission-codes/rust-templa
 - [`tracing-subscriber`][tracing-subscriber] for Rust binary applications
   to collect trace data, such as by logging it to standard output, and
   consume messages emitted by log-instrumented libraries and modules.
+- An option to generate a highly opiniated [`axum`][axum] web application stack.
+
+#### 🔋 Batteries Included Web Framework
+
+If you choose to run a webserver with [`axum`][axum] (choosing `true` at the
+prompt), you'll be given an extensive web framework to work from, heavily
+influenced by [*Composing an observable Rust application*][composing-rust],
+among other sources.
+
+- [`anyhow`][anyhow] as a ergonomic and idiomatic alternative for handling
+  errors in applications.
+- [`axum`][axum] as the fundamenal, already-🔋's included web framework that
+  serves as our foundation, which demonstrates high-level features like:
+  * Routing requests to handlers with a macro free API.
```

Comment:
```suggestion
  * Routing requests to handlers with a macro free API
```

Bullet points don't need trailing punctuation if they are a single sentence. (But keep them to be consistent unless you want to update them all.)